expo/expo
# [expo-constants] Fix Constants.expoConfig for embedded updates

# Why

Adjust `expo-constants` for a fix in the `expo-updates` constants (the manifest) .

# How

Adjust `expo-constants` to read `expoConfig` correctly for cases where `expo-updates` is used, but the app is running from the embedded bundle.

# Test Plan

- Add test for this in Updates E2E
- Check that behavior is correct for embedded updates, downloaded updates, and rollbacks

# Checklist
